content/en/docs/concepts/workloads/pods/pod-lifecycle.md

```diff
@@ -205,31 +205,34 @@ Once Pod is assigned to a node by scheduler, kubelet starts creating containers
     ...
    ```
 
-## Pod readiness gate
+## Pod readiness {#pod-readiness-gate}
 
 {{< feature-state for_k8s_version="v1.14" state="stable" >}}
 
-In order to add extensibility to Pod readiness by enabling the injection of
-extra feedback or signals into `PodStatus`, Kubernetes 1.11 introduced a
-feature named [Pod ready++](https://github.com/kubernetes/enhancements/blob/master/keps/sig-network/0007-pod-ready%2B%2B.md).
-You can use the new field `ReadinessGate` in the `PodSpec` to specify additional
-conditions to be evaluated for Pod readiness. If Kubernetes cannot find such a
+Your application can inject extra feedback or signals into PodStatus:
+_Pod readiness_. To use this, set `readinessGates` in the PodSpec to specify
+a list of additional conditions that the kubelet evaluates for Pod readiness.
+
+Readiness gates are determined by the current state of `status.condition`
+fields for the Pod. If Kubernetes cannot find such a
 condition in the `status.conditions` field of a Pod, the status of the condition
-is default to "`False`". Below is an example:
+is defaulted to "`False`". Below is an example:
```

Comment:
extra nit: Do you need double and single quotes around the boolean value
... the value defaults to `False`